README.md

```diff
@@ -54,13 +52,12 @@ Feel free to check out [issues](https://github.com/encukou/freezeyt/issues) page
 ### How to watch progress
 Unfortunately our progress of development can be watched only in Czech language.
 
-We have been using these channels:
+Watch the progress:
 
-* live streaming on [Jitsi](https://meet.jit.si/freezeyt)
-* videos from live streaming on [Youtube](https://www.youtube.com/playlist?list=PLFt-PM7J_H3EU5Oez3ZSVjY5pZJttP2lT)
-* [Pyvec channel](https://pyvec.slack.com/archives/C0168D5TQ4D) on Slack is `# freezeyt`
-* [document](https://docs.google.com/document/d/1iACdWvSJZmKTgu2TBKYA71jGaiqftDZDqYW6c4BoqjE/edit) in Czech where you can find all information about project
+* [Youtube channel](https://www.youtube.com/playlist?list=PLFt-PM7J_H3EU5Oez3ZSVjY5pZJttP2lT)
 
+Other communication channels and info you will find in:
```

Comment:
```suggestion
You will find other communication channels and info here:
```

Reply:
ja bych nechal to svý jen bych místo in na konci dal tvé here co ty na to

Reply:
Nejsem si jistá s tím *you will find*, nějak mi to tam "nezní" - to už bych tam dala něco jako:
```suggestion
Other communication channels and info can be found here:
```